tests/tomography_test.py

```diff
@@ -45,6 +45,15 @@ def test_binary_int_methods():
     assert binary_to_int(converted_binary) == test_int
 
 
+def test_wire_adding():
+    c = Circuit(10).CX(0, 1)
+    n_qubits = c.n_qubits
+    c.remove_blank_wires()
+    while c.n_qubits < n_qubits:
+        c.add_qubit(Qubit("temp", c.n_qubits))
+    assert c.n_qubits == 10
```

Comment:
Is this test touching any of the Qermit codebase? I think it would be fine not to include this if not. If we are not able to write a test of the Qermit code base then we should be grumpy about it and carry on.

Reply:
I don't mind too much if this stays but I think we more comprehensive tests now.